ansible/roles/debops.nginx/templates/etc/sudoers.d/nginx_webadmins.j2

```diff
@@ -1,7 +1,18 @@
 # {{ ansible_managed }}
 
-Cmnd_Alias NGINX_RELOAD = /etc/init.d/nginx reload
+Cmnd_Alias NGINX_RELOAD = /etc/init.d/nginx reload, \
+			  /bin/systemctl reload nginx.service
```

Comment:
Could you add a space here and in the other similar cases, so that the commands are aligned?

Reply:
Status command added and indentation fixed.